artifactory/services/upload.go

```diff
@@ -30,14 +30,23 @@ import (
 	"github.com/jfrog/jfrog-client-go/utils/log"
 )
 
+const (
+	// 10 KiB
+	DefaultMinChecksumDeploy = utils.SizeKib * 10
+	// 100 MiB
+	defaultUploadMinSplit   = utils.SizeMiB * 200
+	defaultUploadSplitCount = 3
```

Comment:
Let's add descriptions, explaining what "defaultUploadMinSplit" and "defaultUploadSplitCount" represent. For the random reader, who's unfamiliar with those copncepts, it would be unclear.